ios/Classes/AudioRecorder.swift

```diff
@@ -112,20 +112,16 @@ public class AudioRecorder: NSObject, AVAudioRecorderDelegate{
             AVAudioSession.sharedInstance().requestRecordPermission() { [unowned self] allowed in
                 DispatchQueue.main.async {
                     self.hasPermission = allowed
```

Comment:
Can you please remove `hasPermission` variable since it is not required anymore? BTW nice catch.